docker/linux/scripts/configure-tentacle.sh

```diff
@@ -211,6 +214,9 @@ function registerTentacle() {
         if [[ ! -z "$ListeningPort" && "$ListeningPort" != "$internalListeningPort" ]]; then
             ARGS+=('--tentacle-comms-port' $ListeningPort)
         fi
+		if [[ ! -z "$ProxyName" ]]; then
+		  ARGS+=('--proxy' $ProxyName)
```

Comment:
Quote the `$ProxyName` parameter to ensure spaces in the name are captured

```suggestion
		  ARGS+=('--proxy' "$ProxyName")
```